### DISCUSSION TITLE
Store Commerce dual display on 10.0.39 giving error on startup

### DISCUSSION THREAD

**Discussion by PhuMinhPham** [microsoft/Dynamics365Commerce.Solutions#268] [Q&A]
Just upgraded from Store Commerce 10.0.38 to Store Commerce 10.0.39 (9.49.24079.1) and when the dual display is launching, this error pops up and completely breaks the functionality on the dual display
![image](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/64581778/40105295-4c46-4186-bd52-542cc816c13e)

In the DevTools it mentions this error
![image](https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/64581778/1faa73fb-a1f4-4fae-8008-4dbb7a460a78)

Anyone with same or similar issue?

**Comment by zhangguanghuib** [comment]
This dual display solution works for you? https://github.com/zhangguanghuib/NewCommerceSDK/blob/main/KnowledgeArticles-Development/5%20Store%20Commerce%20Dual%20Display%20Support%20AutoScrolling.md
